chore: add post to highlight and AI settings (#94)
open-sauced/docs.opensauced.pizza: docs/chrome-extension/using-the-chrome-extension.md

### Quick Access to important OpenSauced links

![Quick Access to OpenSauced](../../static/img/extension-links.png)

### Posting highlight

> You can access the post to highlight from the main popup in the extension, and when you click on it, it will automatically, get the the PR or issue  title, here you can use our AI functions to get a summary of the highlight similar to the PR summary. 

Reviewer: ```suggestion
You can access the post to highlight it from the popup window in the extension. When you clicked, it will automatically, populate the PR or issue title, here you can use our AI functions to get a summary of the highlight similar to the PR summary. 
```

![location of post to highlight](https://user-images.githubusercontent.com/18273833/241761099-e7a745c8-204e-4a4d-b313-867ab99af0b2.png)

#### Successful highlight:

> If you had a successful highlight it will show you a message with option to see the highlight on the OpenSauced website. 

Reviewer: ```suggestion
If you post a successful highlight it will show you a message with an option to see the highlight on the OpenSauced website. 
```

![Successful message example](https://github.com/open-sauced/docs.opensauced.pizza/assets/18273833/723ac624-e996-45f8-acdf-0d3ff08af90b)

#### Errors example: 

> If you faced an error with the summarization it will be printed in the text area, this is an example of it: 

Reviewer: ```suggestion
If you presented with an error with the summarization it will be printed in the text area, this is an example of it: 
```
